utils/kubernetes/describe/describe.go

```diff
@@ -88,22 +101,30 @@ var ResourceMap = map[DescribeType]schema.GroupKind{
 	EndpointSlice:             {Group: discoveryv1.GroupName, Kind: "EndpointSlice"},
 }
 
+/*
```

Comment:
Replace to `// The Describe() function takes a meshkitkube.Client object and a DescriberOptions object as input.
// And it returns the description of the specified Kubernetes resource as a string.`